### PR TITLE
Added solution for Task01

### PR DESCRIPTION
**Pull request by DrHristov**

### CODE REVIEW

**Review by dejansvetoslavov — APPROVED** [review]

File: SI/Sem.02/Pract.02/Task01.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+int main() {
+	int year = 0;
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/Task01.cpp:4]: Is it me or do we have x2 tabulations ?

File: SI/Sem.02/Pract.02/Task01.cpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	std::cout << "Enter a year: ";
+	std::cin >> year;
+
+	if ((year % 400 == 0) || (year % 4 == 0 && year % 100 != 0)) {
+		std::cout << "Entered year is leap.\n";
+	}
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/Task01.cpp:11]: Suggestion: 
} else {

File: SI/Sem.02/Pract.02/Task01.cpp (hunk 1 above)
Comment (DrHristov) [reply SI/Sem.02/Pract.02/Task01.cpp:4]: It's not just you, I can see them as well. it is definitely not on purpose. There should be only one tabulation. I don't know why that has happened, but in the source file the tabs are single. I've attached a screenshot of the source file that I've uploaded here.
Task01Screenshot

File: SI/Sem.02/Pract.02/Task01.cpp (hunk 2 above)
Comment (DrHristov) [reply SI/Sem.02/Pract.02/Task01.cpp:11]: I completely agree. Changed the setting in VS to automatically put it there. From now on the "else" will be there :D

File: SI/Sem.02/Pract.02/Task01.cpp (hunk 1 above)
Comment (DrHristov) [reply SI/Sem.02/Pract.02/Task01.cpp:4]: I resolved the issue. It's just GitHub that displays the single tab so big that it looks like 2 tabs. But in reality, when copying the code from the file and pasting it in VS the tab is only 1. In conclusion it's just a visual "bug".

**Pull request merged**